### ISSUE TITLE
High Validation accuracy

### ISSUE THREAD

**Issue by testingshanu** [hthuwal/sign-language-gesture-recognition#31]
I was trying to run this script on my own data set with 2 classes containing sequences of images. However, on the second epoch, I am getting high validation accuracy (1.00 ) and low val loss compared to that of training metrics.

Note: Sequence of images are similar to each other.

Any reason for that?

**Comment by hthuwal** [comment]
What is the size of your validation set? Small number validation sample might give you high accuracy but the score is not generalizable.
